### DISCUSSION TITLE
How to direct bypass specific site?

### DISCUSSION THREAD

**Discussion by FilJed** [hiddify/hiddify-app#622] [Q&A]
Hi! 

server settings:  REALITY+VLESS using x-ui on VPS.

Spent a week or so trying to figure out how to bypass specific site (e.g. youtube.com) such that all traffic goes directly to the site, ignoring proxy when proxy is on.

Help me please or point where to look for.
Thx in advance!

**Comment by LitenBuzzTh** [comment]
I guess the hiddify does not have the option to bypass specific domains like v2ray or nekoray. we kindly request the developers to add this in future releases.
